storage/src/adb/benches/variable_init.rs

```diff
@@ -102,8 +102,19 @@ type AnyDb = Any<Context, <Sha256 as Hasher>::Digest, Vec<u8>, Sha256, EightCap>
 fn bench_variable_init(c: &mut Criterion) {
     let cfg = Config::default();
     let runner = tokio::Runner::new(cfg.clone());
-    for elements in [NUM_ELEMENTS, NUM_ELEMENTS * 2] {
-        for operations in [NUM_OPERATIONS, NUM_OPERATIONS * 2] {
+
+    cfg_if::cfg_if! {
```

Comment:
We can set this cfg_if outside the function like the other consts we added? Recommend doing so for consistency?